assets/less/cds-rdm/globals/site.overrides

```diff
@@ -53,4 +53,12 @@
 
 .info-box {
   padding: @defaultPadding !important;
-}
\ No newline at end of file
+
+  a{
```

Comment:
Just the ones in the info-box class.

Reply:
As Manuel pointed this would affect all of them.
Also in general we are trying to be a bit more behaviour based when defining the css.
With this in mind one option could be to change it to:
```
  text-decoration-none:hover{
    text-decoration: none;
  }
  cursor-pointer:hover{
    cursor:pointer
  }
```
This could be defined in invenio-app-rdm as well.
And in the links add both of them 
```
<a href="/search?&f=resource_type:publication" class="four wide column centered cursor-pointer text-decoration-none">
```
WDYT?

Reply:
Discussed IRL. The tag is already inside the info-box selector.

Reply:
What is not clear for me is why are we changing the cursor to pointer since it is already a default for all the `a` tags? Why do we need to redefine it ?